dlp/snippets/src/test/java/dlp/snippets/DeIdentificationTests.java

```diff
@@ -719,4 +720,32 @@ public void testDeIdentifyWithBucketingConfig() throws IOException {
     assertThat(actualTable).isEqualTo(expectedTable);
     assertThat(output).contains("Table after de-identification: ");
   }
+
+  @Test
+  public void testDeIdentifyTableWithCryptoHash() throws IOException {
+
+    Table tableToDeIdentify =
+        Table.newBuilder()
+            .addHeaders(FieldId.newBuilder().setName("userid").build())
+            .addHeaders(FieldId.newBuilder().setName("comments").build())
+            .addRows(
+                Table.Row.newBuilder()
+                    .addValues(Value.newBuilder().setStringValue("user1@example.org").build())
+                    .addValues(
+                        Value.newBuilder()
+                            .setStringValue(
+                                "my email is user1@example.org and phone is 858-555-0222")
+                            .build())
+                    .build())
+            .build();
+
+    // Generate a random string to use it as Transient CryptoKey.
+    UUID uuid = UUID.randomUUID();
+    String randomString = uuid.toString().replace("-", "");
+
+    DeIdentifyTableWithCryptoHash.deIdentifyWithCryptHashTransformation(
+        PROJECT_ID, tableToDeIdentify, randomString);
+    String output = bout.toString();
+    assertThat(output).contains("Table after de-identification: ");
```

Comment:
Please assert that the output doesn't contain the phone number and email address included in the input table.